src/sst/elements/balar/tests/vanadisHandshake/cuda_runtime_api.h

```diff
@@ -157,26 +162,26 @@ typedef struct BalarCudaCallPacket {
             uint64_t dst;
             uint64_t src;
             uint64_t count;
-            enum cudaMemcpyKind kind;
             uint64_t payload;   // A pointer, but need to be 64-bit
+            enum cudaMemcpyKind kind;
         } cuda_memcpy;
 
         struct {
+            uint64_t sharedMem;
+            uint64_t stream;
             uint32_t gdx;
             uint32_t gdy;
             uint32_t gdz;
             uint32_t bdx;
             uint32_t bdy;
             uint32_t bdz;
-            uint64_t sharedMem;
-            uint64_t stream;
         } configure_call;
 
         struct {
             uint64_t arg;
-            uint8_t value[8];
             uint64_t size;
             uint64_t offset;
+            uint8_t value[200];
```

Comment:
@William-An If this is related to the array from above, we should find a way to ensure that this is propagated everywhere that relies on it.